modules/messages/src/mock.rs

```diff
@@ -346,14 +349,30 @@ impl MessageDeliveryAndDispatchPayment<AccountId, TestMessageFee> for TestMessag
 	}
 
 	fn pay_relayers_rewards(
+		lane_id: LaneId,
+		message_relayers: VecDeque<UnrewardedRelayer<AccountId>>,
 		_confirmation_relayer: &AccountId,
-		relayers_rewards: RelayersRewards<AccountId, TestMessageFee>,
+		received_range: &RangeInclusive<u64>,
 		_relayer_fund_account: &AccountId,
-	) {
-		for (relayer, reward) in relayers_rewards {
-			let key = (b":relayer-reward:", relayer, reward.reward).encode();
-			frame_support::storage::unhashed::put(&key, &true);
+	) -> Result<(), Self::Error> {
+		let mut relayers_rewards: RelayersRewards<_, TestMessageFee> = RelayersRewards::new();
```

Comment:
It seems like almost exact copy of `InstantCurrencyPayments::pay_relayer_rewards`, right? Could you, please, make some shared function that prepares the `relayers_rewards` struct && deduplicate the code?